File: CPP/Human.cpp
#include "Human.h"

Human::Human()
{
    this->nik = "";
    this->nama = "";
    this->jenis_kelamin = '-';
}

Human::Human(string nik, string nama, char jenis_kelamin)
{
    this->nik = nik;
    this->nama = nama;
    this->jenis_kelamin = jenis_kelamin;
}

void Human::setNIK(string nik) { this->nik = nik; }
void Human::setNama(string nama) { this->nama = nama; }
void Human::setJenisKelamin(char jk) { this->jenis_kelamin = jk; }

string Human::getNIK() { return this->nik; }
string Human::getNama() { return this->nama; }
char Human::getJenisKelamin() { return this->jenis_kelamin; }

Human::~Human() {}

File: Python/Human.py
class Human:

    __nik = ""
    __nama = ""
    __jenis_kelamin = ''

    def __init__(self, nik="", nama="", jk=''):
        self.__nik = nik
        self.__nama = nama
        self.__jenis_kelamin = jk

    def setNIK(self, nik):
        self.__nik = nik
    
    def setNama(self, nama):
        self.__nama = nama

    def setJenisKelamin(self, jk):
        self.__jenis_kelamin = jk

    def getNIK(self):
        return self.__nik
    
    def getNama(self):
        return self.__nama
    
    def getJenisKelamin(self):
        return self.__jenis_kelamin
    
    